lib/bamboo/adapters/mailgun_adapter.ex

```diff
@@ -20,7 +21,6 @@ defmodule Bamboo.MailgunAdapter do
   """
 
   @service_name "Mailgun"
-  @base_uri "https://api.mailgun.net/v3"
```

Comment:
What do you think about keeping this as a constant here and then referencing the constant below?